cms/core/blocks/section_blocks.py

```diff
@@ -41,15 +44,39 @@ class Meta:
         block_counts: ClassVar[dict[str, dict]] = {"related_links": {"max_num": 1}}
 
 
+class GlossarySectionBlock(StructBlock):
+    title = HeadingBlock(default=_("Definitions"))
+    content = ListBlock(SnippetChooserBlock(GlossaryTerm))
```

Comment:
@zerolab Does Snippets support Multi Select chooser?.

Reply:
Alas that is more for inline models https://docs.wagtail.org/en/stable/reference/panels.html#multiple-chooser-panel